src/domain/Write/History/index.tsx

```diff
@@ -0,0 +1,44 @@
+import dayjs from 'dayjs';
+
+import 'dayjs/locale/ko';
+
+import Icon from '@components/Icon';
+
+import { type WriteHisotry } from '../types';
+import WriteHistoryCard from './components/Card';
+import * as style from './index.css';
```

Comment:
```suggestion
import * as styles from './style.css';
```
엇 이렇게 작성하는 걸로 통일했던 거 같아요.